infrastructure/analytics/analytics.go

```diff
@@ -46,7 +46,6 @@ func SendAnalyticsToAPI(c *config.Config, payload []byte) error {
 
 	if err != nil {
 		logger.Err(err).Msg("error invoking workflow")
-		return err
```

Comment:
Please add a comment why we don't return the error here.

Reply:
LS returns an error the IDEs if analytics-service not present. This should not happen - it should fail silently, with just a log message. As Basti pointed out.